### ISSUE TITLE
Controls screen doesn't open on Steam Deck

### ISSUE THREAD

**Issue by kayahr** [friendlyfiregame/friendlyfiregame#358]
**Describe the bug**
On Steam Deck the Controls screen doesn't show. When selecting this menu item then the game freezes but music is still playing. Game must be killed via Steam Menu.

**To Reproduce**
1. Start game on Steam Deck
2. Select Controls menu.  
3. The top of the screen is darkened but then display freezes. See screenshot.  Music keeps playing but no interaction is possible anymore. Happens in main menu and in pause menu during the game.

**Expected behavior**
Show the correct Controls menu. Don't freeze.

**Screenshots**
![controls](https://github.com/friendlyfiregame/friendlyfiregame/assets/580493/8ad6a666-2f27-4a2d-83e6-0bc7f94dd3bc)

**Execution environment (please complete the following information):**
 - Operating system: Steam Deck
 - Distribution variant: Steam

**Comment by kayahr** [comment]
I enabled electron logging by adding the parameters `--log-file=/tmp/friendly.log --enable-logging` and found this JS error:

```
Uncaught Error: Frame tag not found and fallback is not available as well. Tag: 'unknown' | FallbackTag: 'idle'", source: file:///home/deck/.local/share/Steam/steamapps/common/Friendly%20Fire/resources/app.asar/.webpack/renderer/index.js
```

**Comment by kayahr** [comment]
The problem is the Aseprite `controls_gamepad`. This Aseprite defines the tags `xbox`, `playstation` and `stadia`. But the `ControlsScene` can select the additional tags `unknown` and `snes`. On the Steam Deck it is `unknown` which leads to the thrown exception.

**Comment by kayahr** [comment]
Fixed by 0a555479425b9e5a1a7a8c38247874b2deef0854

I added Valve's Vendor ID to the list so the Steam Deck controller is detected as Xbox Controller. Now the controls menu works and the correct key icons are displayed in game.

But this error can happen again on machines with other unknown controllers.
